taipy/gui/viselements.json

```diff
@@ -1,5 +1,26 @@
 {
     "controls": [
+        [
+            "alert",  
```

Comment:
So you moved the definition of the control to the top.
Actually we use the order of the declaration to generate the documentation.
We want 'more used' elements to appear first.
To me, this should go just after "status".

And same for the properties: more used come first.
In your case the order should be "message", "severity".

Reply:
And where is variant gone?
I guess we can do that with styling, and that's fine.